### PR TITLE
Adds multiple flooring options.

### PR DESCRIPTION
**Pull request by JonahDes**
#### Summary
 ```SUMMARY: Content "Adds flooring, will include new types of terrain in terrain.json"```

#### Purpose of change

This is part of a series of changes that I'm implementing which tries to make the world shapeable by the player. Simple wooden, dirt, and concrete flooring, then moving onwards in future PRs to more complex terrain types.

### CODE REVIEW

File: data/json/construction.json [hunk 1]
```diff
@@ -819,6 +819,37 @@
...
+    "post_terrain": "t_dirt"
+  },
+  {
+    "type": "construction",
+    "description": "Make Woodchip Floor",
+    "//": "Adds wood chippings to a shallowed dirt area to prevent plants growing, or for looks.",
+    "category": "CONSTRUCT",
+    "time": 40,
+    "components": [ [ "splintered_wood", 50 ] ],
+    "pre_terrain": "t_pit_shallow",
```
Comment (I-am-Erk) [inline data/json/construction.json]: You can probably put woodchips down on any dirt area

Comment (JonahDes) [reply data/json/construction.json]: You can. There's usually two kinds, one is a dug box to add depth, and then there's an area just sprinkled with them. In this case I'm fine with it being the standard chip path.

**Comment by AMurkin** [comment]
Summary line should be formatted like so:
`SUMMARY: Content "Adds flooring, will include new types of terrain in terrain.json"`

**Comment by Night-Pryanik** [comment]
> Summary line should be formatted like so:

Done that.

**Comment by AMurkin** [comment]
Jenkins rebuild

**Pull request merged**